idaes/generic_models/properties/core/eos/tests/test_enrtl.py

```diff
@@ -832,3 +833,34 @@ def test_tau(self, model):
         assert ("Cl-", "OH-") not in model.state[1].Liq_tau
         assert ("OH-", "Cl-") not in model.state[1].Liq_tau
         assert ("OH-", "OH-") not in model.state[1].Liq_tau
+
+
+class TestProperties(object):
+    @pytest.fixture(scope="class")
+    def model(self):
+        m = ConcreteModel()
+        m.params = GenericParameterBlock(default=configuration)
+
+        m.state = m.params.build_state_block([1])
+
+        # Need to set a value of T for checking expressions later
+        m.state[1].temperature.set_value(300)
+
+        return m
+
+    @pytest.mark.unit
+    def test_pressure_osmotic_phase(self, model):
+        model.state[1].dens_mol_phase = Var(model.params.phase_list,
+                                            initialize=55e3,
+                                            units=pyunits.mol/pyunits.m**3)
+
+        model.state[1].pressure_osmotic_phase.display()
+        model.state[1].act_phase_comp.display()
+
+        assert_units_equivalent(model.state[1].pressure_osmotic_phase["Liq"],
+                                pyunits.Pa)
+        assert len(model.state[1].pressure_osmotic_phase) == 1
+        assert pytest.approx(value(
+            -Constants.gas_constant*300*log(0.1670306)*55e3),
+            rel=1e-6) == value(
+                model.state[1].pressure_osmotic_phase["Liq"])
```

Comment:
The naming standard I've been using for osmotic pressure was "pressure_osm". I shortened osmotic to osm because saturation pressure in IDAES conventions is "pressure_sat" and I also dropped the phase because I think its only applicable for liquids.

Reply:
Two things there:

1. You didn't update the naming standards, so I didn't know that was your standard.
2. The phase index is still needed because there could be multiple phases.

Reply:
Also as a counter-point, bubble pressure is `pressure_bubble`.

Reply:
Should we be updating IDAES naming conventions for any new names we create for Proteuslib? Or should we have our own section on naming conventions wherever we build on IDAES' naming conventions (or both)?

Reply:
We should try to maintain a single set of standards where possible, so the best approach would be to propose additions to the IDAES standards.

Reply:
I've updated the naming convention and added it to the docs.